bound/kt/src/main/kotlin/web5/sdk/vc/StatusListCredential.kt

```diff
@@ -30,6 +70,41 @@ data class StatusListCredential(
         }
     }
 
+    /**
+     * Checks if a given credential is disabled according to this Status List Credential.
+     *
+     * @param credential The [VerifiableCredential] to check.
+     * @return `true` if the credential is disabled, `false` otherwise.
+     * @throws Web5Exception if there is an error while checking the status of the credential.
+     *
+     * Example usage:
+     * ```
+     * val issuerBearerDid = DidJwk.create(null)
+     * val subjectDidUri = "did:dht:ng4hmqtrgujox4agpf8okxihnyy1zqnq97qfeq15x8oar7yepzhy"
+     *
+     * val verifiableCredential = VerifiableCredential.create(
+     *     Issuer.StringIssuer(issuerBearerDid.did.uri),
+     *     CredentialSubject(id = subjectDidUri),
+     *     VerifiableCredentialCreateOptions(
+     *         credentialStatus = CredentialStatus(
+     *             id = "https://example.com/status/1",
+     *             type = "StatusList2021Entry",
+     *             statusPurpose = "revocation",
+     *             statusListIndex = "3",
+     *             statusListCredential = "https://example.com/status/1"
+     *         )
+     *     )
+     * )
+     *
+     * val statusListCredential = StatusListCredential.create(
+     *     Issuer.StringIssuer(issuerBearerDid.did.uri),
+     *     "revocation",
+     *     listOf(verifiableCredential)
+     * )
```

Comment:
```suggestion
     
```

Since you already created it above in the examples can take this out nad just leave the isDisabled() part